docs/user/modelling.md

```diff
@@ -0,0 +1,128 @@
+# Modelling
+
+This page describes how to model various kinds of fields on a `nautobot_ssot.contrib.NautobotModel` subclass.
+
+## Normal Fields
+
+For normal, non-relationship, non-custom fields on a model, all you need to do is to ensure that field name on your SSoT model class matches that of the Nautobot model class. To ensure this is the case, you can check out the model either by reading the corresponding source code on [GitHub](https://github.com/nautobot/nautobot) or through the [Nautobot Shell](https://docs.nautobot.com/projects/core/en/stable/administration/nautobot-shell/) - in there you can use commands like `dir(Tenant)` to get an overview of available fields for the different models.
```

Comment:
This is one of the reasons I thought the graphql idea was good. Obviously not for this PR, but in general, we need a better story for newcomers to understand how to build this out.

Reply:
Yeah I wasn't happy with this, it requires somewhat intricate knowledge. Would be cool to have a data model published somewhere in a very easy to understand format